### PR TITLE
properly handle imagestreams where number of spec tags will be less t…

### PR DESCRIPTION
**Pull request by gabemontero**
…han number of status tags (when images go EOL)

/assign @adambkaplan

### CODE REVIEW

File: pkg/stub/imagestreams.go [hunk 1]
```diff
@@ -384,7 +384,7 @@ func (h *Handler) processImportStatus(is *imagev1.ImageStream, cfg *v1.Config) (
 		}
 
 		if !anyErrors {
-			if len(is.Spec.Tags) == len(is.Status.Tags) {
+			if len(is.Spec.Tags) <= len(is.Status.Tags) {
```
Comment (adambkaplan) [inline pkg/stub/imagestreams.go]: nit - add a comment to make it clear this covers an upgrade where an upstream tag was removed.

Comment (gabemontero) [reply pkg/stub/imagestreams.go]: comment added @adambkaplan .... figured it was a safe change for me to make given my jetlagged self :-)

**Comment by gabemontero** [comment]
aws flake on e2e-aws

/test e2e-aws

**Comment by gabemontero** [comment]
terraform flake on cluster set up for image eco

/test e2e-aws-image-ecosystem

**Pull request merged**